README.md

```diff
@@ -17,6 +17,36 @@ Compared to LuaJIT, Pallene aims to offer more predictable run-time performance.
 In order to use this source distribution of the Pallene compiler, you need to
 install its Lua library dependencies and compile its run-time library.
 
+### Prerequisites
+
+Before you build Pallene, you need to install `gcc` and `make` on your system.
+Ubuntu users can run the following commands to install these tools.
+```
+sudo apt-get update
+sudo apt-get install build-essential
+```
+
+Linux users using pacman can run the following commands.
+```
+pacman -Syyu
+pacman -S make
+pacman -S gcc
+```
```

Comment:
I think you can remove this part, we could leave the installation instructions just for Ubuntu as an illustration. Specially because Arch users are usually more advanced and know how to install packages by their own.

Reply:
I'll remove it.

PS: You make Arch Linux users look cool! :P